src/components/InlineAlert/InlineAlert.types.ts

```diff
@@ -0,0 +1,14 @@
+import type { ReactElement, ReactNode } from 'react';
+
+export type AlertStatus = 'neutral' | 'informational' | 'error' | 'warning' | 'success';
+
+export interface InlineAlertProps {
+  status?: AlertStatus;
```

Comment:
You can add a comment description for each comment (also renders on the ArgTable or ControlsTable in storybook and enriches the documentation)